classes/OpenXdmod/Setup/AclEtl.php

```diff
@@ -54,9 +54,40 @@ public function execute()
             fwrite(STDOUT, "Command: $cmd\n");
         }
 
-        $output = shell_exec($cmd);
+        $pipes = array();
+
+        $process = proc_open(
+            $cmd,
+            array(
+                0 => array('file', '/dev/null', 'r'), // STDIN : Script doesn't need STDIN
+                1 => array('pipe', 'w'),              // STDOUT
+                2 => array('pipe', 'w')               // STDERR
+            ),
+            $pipes
+        );
+
+        if (!is_resource($process)) {
```

Comment:
Even though the examples don't use it, we should be clear and check for `false` here.
```suggestion
        if ( false === is_resource($process) ) {
```